src/functions/updatePackageVersions.js

```diff
@@ -10,8 +12,23 @@ type Options = {
   cwd?: string
 };
 
+function versionRangeToRangeType(versionRange: string) {
```

Comment:
I'd love this function to be more certain and check the whole regex is correct. `/^\^\d+\.\d+\.\d+$/` for example. It means we can throw/warn when a range someone provided is overwritten (for example they provided `^1.5.0 || ^2.0.0`)

Reply:
I think I'm happy to leave it for now, because that check will be a part of bolts project validation soon.
Same as the check for, do your peerDep and devDeps match. This function assumes they do. We dont currently enforce that, but should soon.

Reply:
Raised https://github.com/boltpkg/bolt/issues/62